File: decompiled/3.18.0/be/ehealth/businessconnector/mycarenet/memberdata/exception/MemberDataBusinessConnectorException.java
package be.ehealth.businessconnector.mycarenet.memberdata.exception;

import be.ehealth.technicalconnector.exception.ConnectorException;
import java.text.MessageFormat;

public class MemberDataBusinessConnectorException extends ConnectorException {
   private static final long serialVersionUID = 1L;

   public MemberDataBusinessConnectorException(MemberDataBusinessConnectorExceptionValues errorCodeValue, Object... params) {
      super(MessageFormat.format(errorCodeValue.getMessage(), params), errorCodeValue.getErrorCode());
   }
}

File: src/main/kotlin/org/taktik/connector/business/memberdata/exception/MemberDataBusinessConnectorException.kt
package org.taktik.connector.business.memberdata.exception

import org.taktik.connector.technical.exception.ConnectorException
import java.text.MessageFormat

class MemberDataBusinessConnectorException(errorCodeValue: MemberDataBusinessConnectorExceptionValues, vararg params: Any) : ConnectorException(MessageFormat.format(errorCodeValue.message, *params), errorCodeValue.errorCode) {
    companion object {
        private val serialVersionUID = 1L
    }
}
